packages/react/package.json

```diff
@@ -35,6 +35,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@digdir/designsystemet-css": "workspace:^",
```

Comment:
Should this be in dependencies? Maybe peerDependencies 🤔 Now it will be installed and never used by us

Reply:
True true, I'll remove it and we'll revisit this at a later date